src/style.css

```diff
@@ -0,0 +1,252 @@
+body {
+  background: linear-gradient(-45deg, #ee7752, #e73c7e, #23a6d5, #23d5ab);
+  background-size: 400% 400%;
+  animation: gradient 15s ease infinite;
+  height: 100vh;
+  font-family: fantasy;
+  margin: 0;
+  padding: 0;
+}
+
+@keyframes gradient {
+  0% {
+    background-position: 0% 50%;
+  }
+
+  50% {
+    background-position: 100% 50%;
+  }
+
+  100% {
+    background-position: 0% 50%;
+  }
+}
+
+nav {
+  display: flex;
+  padding: 20px;
+  background-color: rgba(0, 0, 0, 0.4);
+  height: 60px;
+}
```

Comment:
- [ ] **[OPTIONAL]:** I noticed that you have multiple repeated CSS properties across your stylesheet. It would be great if you can find a way of reducing this duplication of code ♻️. Remember that not repeating pieces of code will keep your codebase clean and readable. 🤓 🛠️ 

From [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md):
![image](https://user-images.githubusercontent.com/77069011/200000947-9e58c4d8-fed4-45f8-a822-7828c6e713f7.png)


In this case, you could do this with flex:

```CSS
.flex {
  display: flex;
}
``` 

And then, use it in your HTML elements where you need a flex container:

```HTML
<div class="flex"> This is a flex container </div>
```

Please, consider fixing this! 🤓